ddp-workspace/projects/ddp-pancan/src/app/components/faq/faq.component.scss

```diff
@@ -0,0 +1,41 @@
+@import '../../../styles/common.scss';
+
+.faq {
+    padding: 4.56rem 1.5rem 6.12rem 1.5rem;
+    text-align: center;
+
+    .title {
+        @extend .static-page-title;
+        margin-top: 0;
+        margin-bottom: 4.56rem;
+    }
+
+    .text {
+        @extend .static-page-paragraph-text
+    }
+
+    .expand-btn {
+        color: #282828;
+        font-family: Untitled-Sans-Medium;
+        font-size: 1rem;
+        letter-spacing: 0;
+        line-height: 20px;
+        padding: 0.63rem 1rem;
+        margin-bottom: 1.56rem;
+        text-transform: uppercase;
+        border: 1px solid #000000;
```

Comment:
you can use mat-stroked-button, it's the same as flat but with border

Reply:
Technically, yes.
However it is used another color for the button (from default material theme that we are using for now)
![image](https://user-images.githubusercontent.com/7396837/122553083-d6313c00-d03f-11eb-8e0e-d791d23deab9.png)

Reply:
Got it! though it still looks ok to override the border-color instead of setup full border style